backend/lib/broadcastWs.js

```diff
@@ -0,0 +1,54 @@
+const url = require('url');
+const { validateJwt } = require('./utils');
+const aws = require('aws-sdk');
+
+const docClient = new aws.DynamoDB.DocumentClient();
+
+
+module.exports.onConnect = (ws, req) => {
+  // Check url
+  const location = url.parse(req.url, true);
+  if (!location.path.startsWith('/room/')) {
+    return ws.close();
+  }
+
+  // Check roomId
+  const roomId = location.pathname.substring(6);
```

Comment:
Do you think we need to name this constant or/and maybe add documentation about what each index is pointing to?